src/components/chatroom/chat/header/header.chatroom.tsx

```diff
@@ -0,0 +1,43 @@
+import React from "react";
+import "./header.chatroom.scss";
+
+type HeaderProps = {
+  id: number;
+  name: string;
+  img: string;
+  userId: number;
+  setUserId: Function;
+};
+
+function Header({ id, name, img, userId, setUserId }: HeaderProps) {
+  let usericon = "/images/profile1.jpeg";
+
+  let username = "";
+
+  if (userId === 0) {
+    username = "박준열";
+    usericon = "/images/profile1.jpeg";
+  } else {
+    username = name;
+    usericon = img;
+  }
```

Comment:
```suggestion
  
  const [userName, setUserName] = useState("");
  const [userIcon, setUserIcon] = useState("/images/profile1.jpeg");

  if (userId === 0) {
    setUserName("박준열");
  } else {
    setUserName(name);
    setUserIcon(img);
  }
```

`userName`과 `userIcon`은 변경되는 데이터이기 때문에 state를 사용하는 게 더 좋을 것 같습니다! 
[중요한 데이터는 변수 말고 리액트 state로 만들랬죠](https://codingapple.com/unit/react-3-state-usestate-hook/)